dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -8542,6 +8542,25 @@ components:
           type: string
         username:
           type: string
+    Profile_PrivateInfo:
+      type: object
+      properties:
+        avatarURL:
+          type: string
+        bio:
+          type: string
+        company:
+          type: string
+        email:
```

Comment:
While re-organizing, I would consider the email private but not really the other stuff. 
At least not on GitHub, but in general we may want to mirror what the source does
i.e. if github makes the field public, we make the field public.